server.js

```diff
@@ -21,8 +21,8 @@
 const http = require('http');

```

Comment:
Consider using a descriptive variable name instead of a generic name like 'PORT'. This can help improve code readability and maintainability.